pages/events/seminar-series.vue

```diff
@@ -27,7 +27,7 @@ export default class SeminarSeriesPage extends QiskitPage {
   upcomingEvents = upcomingSeminarSerieEvents
   pastEvents = pastSeminarSeriesEvents
   hasUpcomingEvents = JSON.stringify(this.upcomingEvents) !== '[]'
-  nextEvent = upcomingSeminarSerieEvents[0] || null
+  nextEvent = this.hasUpcomingEvents ? upcomingSeminarSerieEvents[0] : null
```

Comment:
Good job, @y4izus 

Just one thing. I think that is good to unify the use of `upcomingEvents` instead of using `upcomingSeminarSerieEvents`.